web-test-runner.utils.js

```diff
@@ -207,28 +172,43 @@ systemVariants.forEach((systemVariant) => {
 vrtGroups = [
     ...vrtGroups,
     ...packages.reduce((acc, pkg) => {
-        const skipPkgs = ['bundle', 'modal'];
+        const skipPkgs = ['bundle', 'modal', 'clear-button', 'close-button'];
         if (!skipPkgs.includes(pkg)) {
-            acc.push({
-                name: `vrt-${pkg}`,
-                files: `(packages|tools)/${pkg}/test/*.test-vrt.js`,
-                testRunnerHtml: vrtHTML({
-                    reduceMotion: true,
-                }),
-                browsers: [chromium],
-            });
-            acc.push({
-                name: `vrt-${pkg}-single`,
-                files: `(packages|tools)/${pkg}/test/*.test-vrt.js`,
-                testRunnerHtml: vrtHTML({
-                    systemVariant: 'spectrum',
-                    color: 'light',
-                    scale: 'medium',
-                    dir: 'ltr',
-                    reduceMotion: true,
-                }),
-                browsers: [chromium],
-            });
+            // Check if the package has VRT test files
+            const testDir = pkg.startsWith('tools/')
```

Comment:
added a more explicit check to avoid future issues of components not having vrt test files breaking the test command